scripts/deployment/README.md

```diff
@@ -0,0 +1,13 @@
+# Deployment script
+
+This _Python_ script is there to help deploy DB objects at a clear Postgres database or database server.
```

Comment:
A very simple example of a structure that this script is requiring is absolutely needed I would say